official/transformer/data_download.py

```diff
@@ -416,4 +415,4 @@ def main(unused_argv):
            "closest to the target size (%d)." % _TARGET_VOCAB_SIZE)
 
   FLAGS, unparsed = parser.parse_known_args()
-  main(sys.argv)
+  tf.app.run()
```

Comment:
Since this is a standalone library that download and process the data, tf.app.run() does not do much here. If we want to use all the tf lib here, probably we should change the flag parse part also to absl flags, so that its consistent across the module.